### PR TITLE
Initialized project, created basic classes, xml doc, etc.

### LINKED ISSUE #16: Init project
* create WPF project/solution called 'task2'
* create classes `Point` and `Pentagon` and figure out how to serialize/deserialize them into XML (the next task will be to create service class that does that)
* the pentagon should have five points and a color property. Read more about how to draw a pentagon here: https://stackoverflow.com/questions/19219544/how-do-i-draw-polygon-in-c-sharp-wpf-project

### PR DESCRIPTION
**Pull request by vkhoroshchak**
Resolves #16

### CODE REVIEW

File: task2/Task2TeamNull/Color.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+﻿using System.Xml.Serialization;
+
+namespace Task2TeamNull
+{
+    public class Color
+    {
+        /// <summary>
+        /// Gets/Sets an amount of red colour in RGB specification
```
Comment (ojles) [inline task2/Task2TeamNull/Color.cs]: Change comment to "Red color intensity".
Change this for Green and Blue

File: task2/Task2TeamNull/Color.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        public byte G { get; set; }
+
+        /// <summary>
+        /// Gets/Sets an amount of blue colour in RGB specification
+        /// </summary>
+        [XmlAttribute]
+        public byte B { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Color"/> class.
```
Comment (ojles) [inline task2/Task2TeamNull/Color.cs]: Remove " class." ending

File: task2/Task2TeamNull/Color.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        [XmlAttribute]
+        public byte B { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Color"/> class. 
+        /// </summary>
+        public Color() { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Color"/> class
```
Comment (ojles) [inline task2/Task2TeamNull/Color.cs]: Remove `e <see cref="Color"/> **class**` ending

File: task2/Task2TeamNull/Color.cs [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        /// Function to set colour of the border using RGB specification
```
Comment (ojles) [inline task2/Task2TeamNull/Color.cs]: Remove this incorrect comment

File: task2/Task2TeamNull/Color.cs [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        /// </summary>
+        /// <param name="r">Amount of red</param>
```
Comment (ojles) [inline task2/Task2TeamNull/Color.cs]: **Intensity** of red

File: task2/Task2TeamNull/pentagon.cs [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        /// Points of pentagon vertices
+        /// </summary>
+        public Point A { get; set; }
+        public Point B { get; set; }
+        public Point C { get; set; }
+        public Point D { get; set; }
+        public Point E { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Pentagon"/> class.
```
Comment (ojles) [inline task2/Task2TeamNull/pentagon.cs]: Remove " class." ending

File: task2/Task2TeamNull/pentagon.cs [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Pentagon"/> class.  
+        /// </summary>
+        public Pentagon()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Pentagon"/> class.
```
Comment (ojles) [inline task2/Task2TeamNull/pentagon.cs]: Remove " class." ending

File: task2/Task2TeamNull/point.cs [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+    /// <summary>
+    /// Class to represent a point in two-dimensional space
+    /// </summary>
+
+    [Serializable]
+    public class Point
+
+    {
+        /// <summary>
+        /// Gets/Sets X coordinate of a point
```
Comment (ojles) [inline task2/Task2TeamNull/point.cs]: Just leave "X coordinate of a point", the same goes for Y and Z

**Review by ojles — CHANGES_REQUESTED** [review]
Fix small issues

**Comment by ojles** [comment]
@Gumbew Also don't forget to rename:
point.cs => Point.cs
pentagon.cs Pentagon.cs

Create a directory `Domain` and move Point.cs, Pentagon.cs, Color.cs into this dir

File: task2/Task2TeamNull/point.cs (hunk 8 above)
Comment (vkhoroshchak) [reply task2/Task2TeamNull/point.cs]: This is a StyleCop requirement
This is a StyleCop requirement

**Review by ojles — APPROVED** [review]

**Pull request merged**